pandasai/llm/langchain.py

```diff
@@ -11,7 +11,25 @@ class LangchainLLM(LLM):
     _langchain_llm = None
 
     def __init__(self, langchain_llm):
-        self._langchain_llm = langchain_llm
+        try:
+            from langchain.llms.base import BaseLLM
+
+            print("*" * 100)
+            print(langchain_llm)
+            print("*" * 100)
```

Comment:
Beware of this "debugging" leftovers 😆

Reply:
Lol the rabbit had promptly complained about it too! 😛